src/home/homed-home.h

```diff
@@ -0,0 +1,168 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+typedef struct Home Home;
+
+#include "homed-manager.h"
+#include "homed-operation.h"
+#include "list.h"
+#include "ordered-set.h"
+#include "user-record.h"
+
+typedef enum HomeState {
+        HOME_UNFIXATED,               /* home exists, but local record does not */
+        HOME_ABSENT,                  /* local record exists, but home does not */
```

Comment:
Maybe call those two HOME_RECORD_MISSING and HOME_STORAGE_MISSING?

Reply:
hmm, so i used "unfixated" here, to match the "fixating" states.

I am not sure I like the "missing" word in this context though, as that sounds as something was wrong. But it really isn#t in either of the cases, i.e. "unfixated" really just means we haven#t fixated the thing yet, i.e. copied stuff from the home to the local system. Hence, nothing is missing at all, it's just that we didn't "instantiate" the thing for use on the local system yet. Similar for "absent": there's nothing missing here, it's just the at the very moment you can't log in because maybe the user removed the USB stick with his home directory on, but as soon as he plugs it in, all is good. 

i.e. "missing" sounds too much of an error condition too me, but this is really not an error at all...